jekyll/_cci2/config-policy-management-overview.adoc

```diff
@@ -22,6 +24,11 @@ CircleCI uses `config.yml` files to define CI/CD pipelines at the project level.
 
 Config policy decisions are stored, and can be audited. This provides useful data about the pipeline definitions being run within your organization.
 
+[#quickstart]
+== Quickstart
+
+If you already have a link:/docs/first-steps[CircleCI account] hooked up with your VCS, and you would like to get started with config policy management right away, check out the steps in the link:/docs/use-the-cli-and-vcs-for-config-policy-management/#create-and-upload-a-policy[Use the CLI and VCS for Config Policy Management] how-to guide.
```

Comment:
I'm also wondering if we can start using `xref` instead of `link` for internal links? I'm not sure if it's still buggy, but I've been able to use `xref` both locally and in production successfully lately.

```suggestion
If you already have a link:/docs/first-steps[CircleCI account] connected to your VCS, and you would like to get started with config policy management right away, check out the steps in the link:/docs/use-the-cli-and-vcs-for-config-policy-management/#create-and-upload-a-policy[Use the CLI and VCS for config policy management] how-to guide.
```

Reply:
I think the issue with xref is that it's not working when we have pages in subdirectories (like the server 4 docs) but works fine for pages just in `_cci2`. But we should revisit and test again. I agree it would be much better to get back to using xref as a best practice. I'll make a ticket to investigate.